notebooks/official/matching_engine/sdk_matching_engine_create_stack_overflow_embeddings_palm.ipynb

```diff
@@ -0,0 +1,2028 @@
+{
```

Comment:
<p>drop "will"</p><p> </p><p>perhaps some explanation on how you prepare the data</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1876/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>Agree, and maybe add an article -- "Use the Stack Overflow dataset..."</p>

Reply:
<p>Done</p>